css/styles.css

```diff
@@ -1,850 +1,1456 @@
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    gap: 2rem;
+    border: none;
+  }
+
+  .form-text {
+    width: 375px;
+    height: 48px;
+    margin-left: 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    border: 0;
+    padding: 0;
+    border-bottom: 1px solid #888;
+  }
+
+  .form-msg {
+    height: 114px;
+    background: #fbf8f7;
+    width: 375px;
+  }
+
+  #form-submit {
+    width: 131px;
+    height: 40px;
+    background-color: #ff6b00;
+    margin: auto;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    padding-left: 15px;
+  }
+
+  #form-submit:hover {
+    background-color: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  #form-submit:active {
+    background-color: #e05304;
+    color: #f4f1f0;
+  }
+
+  #form-submit:disabled {
+    background-color: #f4f1f0;
+    color: #5b5957;
+  }
+
+  .form-icon-container {
+    justify-content: center;
+    align-items: center;
+    padding: 10px;
+  }
+
+  .form-icon-element {
+    background-color: #fff;
+  }
+
+  hr {
+    border-top: 1px solid #888;
+    width: 100%;
+    margin-bottom: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .header-container {
+    display: flex;
+    background: #3e3c3c;
+    align-items: center;
+    justify-content: center;
+    gap: 1.5%;
+  }
+
+  .header-item4 {
+    display: none;
+  }
+
+  .header-item3 {
+    width: 50%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    color: #fff;
+    text-decoration: none;
+    padding-left: 5%;
+  }
+
+  .header-item5,
+  .header-item6,
+  .header-item7,
+  .header-item8 {
+    width: 7%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .intro-section-container {
+    width: 100%;
+    height: 870px;
+    background: #1c1a19;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    background-image: url(../src/bg2.png);
+    background-repeat: no-repeat;
+    background-size: 40% 100%;
+    background-position: right;
+  }
+
+  .intro-content {
+    width: 35%;
+  }
+
+  .icon-container {
+    width: 10%;
+    list-style: none;
+    margin: auto 0;
+    line-height: 50px;
+    padding: 0;
+  }
+
+  .icon-element {
+    padding-left: calc((100% - 18px) / 2);
+  }
+
+  #intro-text1 {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
```

Comment:
- [ ] Kindly try to match the all **fonts** in your design with the [Figma design](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278&scaling=min-zoom&page-id=1%3A1471&hide-ui=1) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏 😂 

example
**N** | **Your design** | **Figma design**
| --- | --- | --- |
1 | ![image](https://user-images.githubusercontent.com/102757126/199937366-086219ce-f5ed-4548-94b1-20df05a668db.png) | ![image](https://user-images.githubusercontent.com/102757126/199937510-2fdb78b3-ffeb-41cd-bd1c-59c5a97cc104.png)
2 | ![image](https://user-images.githubusercontent.com/102757126/199938047-b1e79f93-c009-4bac-8615-6811fad1d8c8.png) | ![image](https://user-images.githubusercontent.com/102757126/199938524-4868a7d8-c5c9-42c5-a6e1-408b2b7feddd.png)
3 | ![image](https://user-images.githubusercontent.com/102757126/199938693-d7a4464c-21f2-419a-8268-e08fb2cba11e.png) | ![image](https://user-images.githubusercontent.com/102757126/199939000-3619d32d-a9cc-4929-bc84-b515c70bf040.png)
5 | ![image](https://user-images.githubusercontent.com/102757126/199939764-5ebd7ef9-1a41-44b8-bf94-722fe0885444.png) | ![image](https://user-images.githubusercontent.com/102757126/199940148-ab51eab5-1fd6-481e-b297-b1dc101c1c99.png)